src/GregClient/Utility/Utilities.cs

```diff
@@ -190,52 +187,53 @@ public static string Relaxed(string value)
         }
     }
 
-    public class StringValueAttribute : System.Attribute
-    {
-
-        private string _value;
 
-        public StringValueAttribute(string value)
+    
+    public static class RestSharpExtensions
+    {
+        /*
```

Comment:
are you planning on continuing this work in a later PR ?

Reply:
Unclear, for now I will remove the commented out code from this PR.

Reply:
I think the whole class can be made internal